test/contestant_test.rb

```diff
@@ -0,0 +1,54 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/contestant'
+require './lib/game'
+
+
+class ContestantTest < Minitest::Test
+  def test_it_exists
+    alexander = Contestant.new({first_name: 'Alexander',
+                                last_name: 'Aigiades',
+                                age: 28,
+                                state_of_residence: 'CO',
+                                spending_money: 10})
+    assert_instance_of Contestant, alexander
+  end
+
+  def test_it_has_attributes
+    alexander = Contestant.new({first_name: 'Alexander',
+                                last_name: 'Aigiades',
+                                age: 28,
+                                state_of_residence: 'CO',
+                                spending_money: 10})
+
+    assert_equal "Alexander Aigiades" ,alexander.full_name
+    assert_equal 28, alexander.age
+    assert_equal "CO", alexander.state_of_residence
+    assert_equal 10, alexander.spending_money
+  end
+
+  def test_out_of_state?
+    alexander = Contestant.new({first_name: 'Alexander',
+                                last_name: 'Aigiades',
+                                age: 28,
+                                state_of_residence: 'CO',
+                                spending_money: 10})
+    assert_equal false, alexander.out_of_state?
```

Comment:
We always want to test both a true and false return value for our boolean methods. This will ensure that the condition is setup correctly.